css-bootstrap/styles/sass/styles.scss

```diff
@@ -0,0 +1,93 @@
+@import "variables";
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  font-family: 'Poppins';
+  background-color: #FFFFFF;
+}
+
+.prueba {
+  width: 80vw;
+}
+
+.overflow-x-auto::-webkit-scrollbar {
+    display: none;
+}
+
+.text-transparent-bgclip {
+  background: linear-gradient(46.99deg, #0F58B7 -4.4%, #62D9FF 120.42%);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  background-clip: text;
+}
+
+.text-fill-black {
+  -webkit-text-fill-color: #1C1C1C;
+}
+
+.border-box {
+  transform: translateY(50%);
+  border: 0.9px solid rgba(0, 0, 0, 0.03);
+  box-shadow: 0px 1px 10px rgba(96, 128, 153, 0.1);
+  border-radius: 5px;
+}
+
+.m-inline {
+  margin-inline: auto;
+}
```

Comment:
this name should be m-inline-auto to indicate what the class is actually doing, and not only m-inline since that's the name of the property